src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -88,9 +91,16 @@ class CrashHeader extends React.Component {
     const {title, beforeTitle, hideGuide, stackView, stackType, newestFirst} = this.props;
 
     let titleNode = (
-      <h3 className="pull-left">
+      <StyledH3
+        className="pull-left"
+        style={{
+          marginBottom: 0,
+          maxWidth: '100%',
+          whiteSpace: 'nowrap',
+        }}
```

Comment:
Do you need to keep in line styles if you also have a styled component?

Reply:
yes.. some CSS properties have a higher specificity because of nested rules and I didn't want to use` !important`

Reply:
Yeah I'm not sure what the right solution is for this. my initial thoughts are just to burn it down all together.

Reply:
Blerg, inline styles work well then :cry: